mergify_engine/json_enum.py

```diff
@@ -0,0 +1,49 @@
+# -*- encoding: utf-8 -*-
```

Comment:
We want to support more than just `Enum` in the future, so that should just be `json` not `json_enum`.